src/App.jsx

```diff
@@ -27,23 +29,54 @@ import Team from "./views/Workspace/Sections/Team";
 
 
 const mapStateToProps = (state) => ({
+  playgrounds: state.playgrounds.playgrounds,
```

Comment:
You should not (ever) directly access the state like this. So the selector provided by the reducer modules instead. In this case you do: `playgrounds: getAllPlaygrounds(state)` . (imported from PlaygroundReducer)

Reply:
Oh, sure, ok. I got it. Thanks!